config/routes.rb

```diff
@@ -76,19 +76,20 @@
             get :summary
           end
         end
-      
+
         resources :webhooks, only: [] do
           collection do
             post :chargebee
           end
         end
       end
-
     end
   end
 
   scope module: 'mailer' do
```

Comment:
let's get rid of this scope. we don't need it.  
Also, add a comment that these routes are to be used in mailers.